File: C/URI/1008.c
#include <stdio.h>
 
int main() {
 
int num, horas_trab;
  double valor_hora, total;
    scanf("%d %d %lf", &num, &horas_trab, &valor_hora);
    total = valor_hora * horas_trab;
    printf("NUMBER = %d\n", num);
    printf("SALARY = U$ %.2lf\n", total);
 
    return 0;
}

File: JS/URI/1008.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var num = parseInt(lines.shift());
var salario = parseInt(lines.shift());
var valor = parseFloat(lines.shift());
var total = salario*valor;
console.log("NUMBER =", num);
console.log("SALARY = U$", total.toFixed(2));

